CMakeLists.txt

```diff
@@ -60,6 +64,12 @@ set(LUAU_OPTIONS)
 if(MSVC)
     list(APPEND LUAU_OPTIONS /D_CRT_SECURE_NO_WARNINGS) # We need to use the portable CRT functions.
     list(APPEND LUAU_OPTIONS /MP) # Distribute single project compilation across multiple cores
+
+    if(LUAU_STATIC_CRT)
```

Comment:
Would be cleaner to define it in a separate top-level if block `if(LUAU_STATIC_CRT AND MSVC)`, so that here we simply set `LUAU_OPTIONS` without global setup